src/main.rs

```diff
@@ -169,6 +233,123 @@ fn rocket() -> rocket::Rocket {
     };
 
     rocket::ignite()
-        .mount("/", routes![index, health])
+        .mount("/", routes![index, css_table_index, health])
         .register(catchers![not_found])
-}
\ No newline at end of file
+}
+
+#[cfg(test)]
+mod tests {
+    use super::{css_table_highlight, CSSTableQuery};
```

Comment:
Why should these tests be in the main file and not in the `src/classed_table_generator.rs` file? This seems odd

Reply:
Because they are testing the `css_table_highlight` function, which lives in here. I can move that to the other file along with the tests -- probably better organized there anyways.